src/s2i/bin/run

```diff
@@ -21,6 +21,9 @@ elif [ "${VERSION}" == "varnish-5" ]; then
 elif [ "${VERSION}" == "varnish-6" ]; then
   PORT_EXPOSE="-a :8080 -a :8443"
   USER_GROUP='-j unix,user=default,ccgroup=root'
+elif [ "${VERSION}" == "varnish-7" ]; then
```

Comment:
Shouldn't we rather use one condition combining 6 and 7 when the block inside is the same?

Reply:
Yeah, by the next PR we will remove deprecated functions, like Varnish-5 and Varnish-4

Reply:
Fixed by https://github.com/sclorg/varnish-container/pull/115/commits/aea3391316055858a2e5b9895820baa0b4c827e6